### ISSUE TITLE
Sort

### ISSUE THREAD

**Issue by wi1d5ky** [isdampe/Notedown#9]
I think that it's more convenient if we could sort article by our will?

**Comment by isdampe** [comment]
Yes sir, absolutely. Drag and drop would be nice. There's a nice library here to handle resorting of lists (could be modified)

https://github.com/bgrins/nativesortable

**Comment by wi1d5ky** [comment]
Drag and drop! It's fascinating!
I thought about 'sort by date', 'sort by title', ... etc.
But drag and drop is better!
